data-transfer-pipeline/extract.py

```diff
@@ -3,43 +3,41 @@
 from os import environ as ENV
 
 import pandas as pd
-import pyodbc
+import pymssql
```

Comment:
We don't like you anymore pyodbc, pymssql is our friend now

Reply:
yes